packages/orchestrator-ui-components/src/components/WFOPageTemplate/WFOSidebar/WFONewSubscriptionButtonComboBox.tsx

```diff
@@ -0,0 +1,43 @@
+import { useRouter } from 'next/router';
+import { useQueryWithGraphql } from '../../../hooks';
+import { GET_WORKFLOWS_FOR_DROPDOWN_LIST_GRAPHQL_QUERY } from '../../../graphqlQueries/workflows/workflowsQueryForDropdownList';
+import { PATH_NEW_SUBSCRIPTION } from '../paths';
+import React from 'react';
+import {
+    ComboBoxOption,
+    WFOButtonComboBox,
+} from '../../WFOButtonComboBox/WFOButtonComboBox';
+import { useTranslations } from 'next-intl';
+
+export const WFONewSubscriptionButtonComboBox = () => {
```

Comment:
Same comments as above about the naming. I think this should be WFONewProcessButtonComboBox

Reply:
Renamed to WFOStartCreateWorkflowButtonComboBox